kafkakewl-domain/src/main/scala/com/mwam/kafkakewl/domain/Deployments.scala

```diff
@@ -93,3 +95,5 @@ object DeploymentsFailure {
   *   the statuses of the topology deployments.
   */
 final case class DeploymentsSuccess(statuses: Map[TopologyId, TopologyDeploymentStatus])
+
+case class TimeoutException[E](exception: E)
```

Comment:
If this goes away it does not matter, but if it stays, it should be in another file.